scripts/tests/test_transition_metrics.py

```diff
@@ -310,6 +310,60 @@ def test_commodity_to_prototype4(self):
             transactions_df, 'tails', 'Reactor_type3')
         assert_frame_equal(exp, obs[0:4])
 
+    def test_commodity_to_LWR1(self):
+        '''
+        This function tests the transactions returned when the 
+        commodity specified is used in the simulation and the 
+        prototype specified is in the simulation and receives
+        the commodity specified. 
+        '''
+        exp = pd.DataFrame(
+            data={
+                'Time': [
+                    0, 1, 2, 3], 'Quantity': [
+                    0.0, 99000.0, 33000.0, 0.0], 'Year': [
+                    1965.00, 1965.08, 1965.17, 1965.25]})
+        transactions_df = tm.add_receiver_prototype(self.output_file1)
+        obs = tm.commodity_to_LWR(
```

Comment:
It looks like the commodity is `fresh_uox` and the prototype is `Reactor_type2`. Why isn't this specified in the docstring?